Remnant: Cognizance 1 (#4871)
endless-sky/endless-sky: data/remnant/remnant missions.txt

			`	"Here, look!" He rewinds the recording and gestures curtly at the projection in front of you. "There. That one is injured. Some kind of burn in a long, straight line." He zips through the footage farther "and here, a scattering of scorch marks." He looks up at you and snaps out a curt series of gestures: "This is outside my experience, I need to consult an expert. Meet me in the spaceport in an hour."`

mission "Remnant: Cognizance 2"
	name "Eliminate Korath Ship"
	description "Plume would like you to eliminate the Korath ship in the Nenia system."
	source "Aventine"
	waypoint
		system "Nenia"
	to offer
		has "Remnant: Cognizance 1: done"
	on offer
		conversation
			`You arrive in the spaceport to find Plume pacing quickly along a walkway. He gestures a quick greeting and jumps straight to the point. "We have never seen marks like these naturally appear on void sprites, but they match the weapon scoring patterns we have seen delivered by Korath ships. As such, we believe the Korath are hunting the void sprites."`

> 			`	He looks at you with a surprising amount of vigor. "I will not have our work ruined by reptilian warmongers. Destroy the Korath ship, then we can continue the research."`

Author: That's a good idea! I have added a conversation here wherein the player has the choice to either accept the mission or challenge Plume's authority. From the Challenge option one gets more information on how the Remnant divy up military authority, with a few options to push for more info and find out a bit about the prefects too.

				accept
	npc
		government "Korath"
		personality staying
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath Raider (Ember)" 3
	on complete
		conversation
			`As you descend on Aventine, Plume gestures towards the pad closest to his lab. "You might as well set down there. If the samples in my lab check out, I will have another mission for you right away. You will need to keep your high-pressure ship active for this one. Stop by the lab in an hour or two and we will be ready to start loading."`

mission "Remnant: Cognizance 8"
	name "Transport engineer to Esquiline"
	description "An engineer working for Plume needs transportation for himself and some equipment to Esquiline."
	blocked "Plume's engineer needs a free bunk and 8 tons of cargo space for his equipment. Come back when you have enough space."
	cargo equipment 8
	passengers 1
	source "Aventine"
	destination "Esquiline"
	to offer
		has "Remnant: Cognizance 7: done"
	on offer
		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed esquiline station.`
		conversation

> 			`You walk into Plume's lab to be greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`

Reviewer: ```suggestion
			`You walk into Plume's lab and are greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`
```

> 			`	"It will be a pleasure to work with you" sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`

Reviewer: ```suggestion
			`	"It will be a pleasure to work with you," sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`
```
Not sure "askance" is the word you're looking for here as it means "with suspicion or distrust."

				accept
	on complete
		payment 100000
		conversation

> 			`Gavriil gives you precise directions to a co-ordinate that your sensors tell you is an empty area, and when you reach the designated spot, you see a small station shimmering out of the clouds. "We never managed to actually cloak the station," grins the engineer. "But we managed to make it hide very effectively against every form of scanning we knew of. We even managed to give it a cloak-like field that renders it mostly transparent."`

Reviewer: ```suggestion
			`Gavriil gives you precise directions to a set of coordinates that your sensors tell you is an empty area, and when you reach the designated spot, you see a small station shimmering out of the clouds. "We never managed to actually cloak the station," grins the engineer. "But we managed to make it hide very effectively against every form of scanning we knew of. We even managed to give it a cloak-like field that renders it mostly transparent."`
```

			`	He nods approvingly. "Rather impressive what my predecessors managed to do with what they had back then."`
			choice
				`	"What do you mean?"`
				`	"Let's get on with it."`
					goto skipinfo
			`	"Well, this place is fairly old. It was built shortly after we learned how to build Starlings where we learned of the troubles of extending the cloaking field beyond small ships. I don't know the mechanics behind it, but it has taken close to two centuries of study and development for them to get cloaks to the point where they are feasible on large ships." We had plenty of puffins back then, and the techniques for building high-pressure systems were well understood. The Alphas hadn't shown any interest in gas giants or 'gaslined ships' as we call them, so having somewhere like this to hide in makes sense. Best place to hide is somewhere no-one will look, right?." With that, he nods to himself, climbs to his feet, and heads towards the suit locker.`
			label skipinfo

> 			`	"The station should be pressurized with a normal atmosphere, but it has been running on standby for ages with just yearly maintenance. It is time to get this place up and running again." With two of you running the equipment, everything is unloaded in short order. You notice that the last crate includes a substantial amount of field rations and survival gear. It looks like he is planning on living rough for a while.`

Reviewer: ```suggestion
			`	"The station should be pressurized with a normal atmosphere, but it has been running on standby for ages with just yearly maintenance. It is time to get this place up and running again." With two of you running the equipment, everything is unloaded in short order. You notice that the last crate includes a substantial amount of field rations and survival gear. It looks like Gavriil is planning on staying here for a while.`
```

mission "Remnant: Cognizance 9"
	name "Pick up a technician and more supplies and deliver to Esquiline."
	description "You need to pick up a technician and their equipment from Caelian and take them to Esquiline."
	blocked "Plume's technician needs a free bunk and 8 tons of cargo space for his equipment. Come back when you have enough space."
	cargo equipment 8
	passengers 1
	landing
	source "Esquiline"
	stopover "Caelian"
	to offer
		has "Remnant: Cognizance 8: done"
	on offer
		dialog
			`As you look around this earily silent station you get a message on your comlink: Plume requests that you pick up another technician and 8 tons of equipment from Caelian and bring it back to Esquiline. `
				accept
	on stopover
		dialog
			`Flight control gives you directions to a landing pad that already has a large pile of crates stacked next to it. As you land, workers swarm out to help load the equipment. Once everything is stowed, a young technician climbs aboard, stows their gear, and lets you know that everything is ready for departure. You shake your head, impressed at how quickly the Remnant make things happen when projects start to move.`
	on complete
		payment 100000
		conversation

> 			`On the flight over, the technician joins you on the control deck to chat. She's a biotech specialist, adept at managing life-support systems in alien atmospheres, and a recent graduate from a Remnant educational program that sounds kind of like university. Apparently she studied astrobiology, and is part of a project aimed at supporting mutually exclusive environments in close proximity. She is eager to be taking part in this current operation, as working on the systems for a covert station are a good test for her skills.`

Reviewer: Hmm, "mutually exclusive environments" doesn't sound right.

> She is eager to be taking part in this current operation, as working on the systems for a covert station are a good test for her skills.`

Show, don't tell. :)

Author: It is kind of a weird situation, but then again, supporting an Oxygen/Nitrogen/CO2 atmosphere inside a predominantly Hydrogen/Helium atmosphere. Or whatever Esquiline's atmosphere actually is.

			`	The <ship> settles into the landing bay, and the technician quickly disembarks. Apparently there must have been other shipments while you were gone, as there are now a handful of people visible and quite a bit more equipment stacked everywhere. The station itself feels more alive, the air fresher. As before, you promptly receive notification of another payment of <payment> to your account.`

mission "Remnant: Cognizance 10"
	name "Remnant Data Delivery"
	description "Deliver a datacube to the engineers on Viminal."
	landing
	source "Esquiline"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 9: done"
	on offer
		conversation

> 			`As you finish unloading you spot Gavriil approaching. He gestures in greeting as he approaches. "Thanks for the delivery. We've got standard logistics routes set up now for the rest."  He pauses and fishes a datacube out of a pouch. "Could you take this to Prefect Taely on Viminal? She is overseeing some retrofits and will need this data.`

Reviewer: ```suggestion
			`As you finish unloading, you spot Gavriil approaching. He gestures in greeting as he approaches. "Thanks for the delivery. We've got standard logistics routes set up now for the rest."  He pauses and fishes a datacube out of a pouch. "Could you take this to Prefect Taely on Viminal? She is overseeing some retrofits and will need this data.`
```

mission "Remnant: Cognizance 11"
	name "Travel with the <escort target ship> to pick up a shipment of atmosphere from Nasqueron and deliver it to <destination>."
	description "Go meet up with escorts"
	cargo equipment 8
	source "Viminal"
	stopover "Nasqueron"
	destination "Esquiline"
	npc accompany
		government "Remnant"
		personality timid launching escort
		fleet
			names "remnant"
			cargo 8 equipment
			variant
				Pelican
				Puffin 4
	to offer
		has "Remnant: Cognizance 10: done"
	on offer
		conversation
			`In the cafeteria, you are met by Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with a cargo of equipment designed to harvest some atmosphere from the void sprite's worlds before bringing it back to Esquiline. The equipment on the puffins will collect, compress, and store the gas, which you can offload to specially prepared tanks on the pelican. Then proceed to Esquiline and offload what you have collected to the station.`
				accept
	on stopover
		dialog

> 			`You cruise through the atmosphere at a relaxed pace, waiting while the equipment does its work. You see fewer Void Sprites than usual, and many have scars. Once the equipment indicates that it has reached capacity you climb up to where the pelican is waiting in the high atmosphere, hook up the connections to offload the gas, then head back down into the depths of the planet. After several hours the captain of the pelican informs you that the tanks are full, and it is time to head to Esquiline.`

Reviewer: ```suggestion
			`You cruise through the atmosphere at a relaxed pace, waiting while the equipment does its work. You see fewer Void Sprites than usual, and many have scars. Once the equipment indicates that it has reached capacity, you climb up to where the pelican is waiting in the high atmosphere, hook up the connections to offload the gas, and then head back down into the depths of the planet. After several hours, the captain of the pelican informs you that the tanks are full, and it is time to head to Esquiline.`
```

	on complete
		event "remnant: postverta reveal"
		conversation

> 			`You arrive at the hidden station on Esquiline, where Gavriil is waiting to offload the atmospherice gasses. His team quickly hooks up the puffin up to ports on the bay walls and the equipment starts ejecting its contents into the system. After a few minutes, they disconnect the ships, and you and the puffins head up to the pelican to refill your reservoirs and ferry it down to the station.`

Reviewer: ```suggestion
			`You arrive at the hidden station on Esquiline, where Gavriil is waiting to offload the atmospheric gasses. His team quickly hooks the puffin up to ports on the bay walls, and hidden equipment starts pumping its contents into the system. After a few minutes, they disconnect the ships, and you and the puffins head up to the pelican to refill your reservoirs and ferry it down to the station.`
```

			distance 1408
			period 554
			sprite planet/sheragi_postverta
		object
			distance 1208
			period 654
			sprite planet/rock10
		object
			distance 439
			period 143
			sprite planet/desert7
		object
			distance 814
			period 362
			sprite planet/fog0
			object
				distance 202
				period 21
				sprite planet/dust4
		object
			period 19
			sprite star/m4
		pos 297 269
	unvisit Levana
	unvisit Caeculus
	unvisit Prosa
	unvisit Egeria
	system Levana
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Caeculus
		add object wormhole_barren2
			distance 2400
			period 391
			sprite planet/wormhole-red
	system Prosa
		add object wormhole_barren2
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Egeria
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
		

mission "Remnant: Cognizance 12"
	name "Investigate strange sensor readings"
	description "Long range sensors have picked up strange gravitational waves. Go investigate."
	landing
	source "Viminal"
	waypoint
		system Postverta
	stopover
		planet "Ssil Vida"
	to offer
		has "Remnant: Cognizance 11: done"
	on offer
		conversation

> 			`You land by the starport and is immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship before escorting you to a nearby starling. Inside you find Chilia and several other Remnants clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match normal star maps.`

Reviewer: ```suggestion
			`You land by the starport and are immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship and escorts you to a nearby starling. Inside you find Chilia and several other Remnants clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match the star maps you're familiar with.`
```

> 			`	Chilia notices you eyeing the deviations. "Perceptive, captain. Our sensors have reported some unusual energy signatures coming from this location. Scouts in the area says they are picking up signs of a new system. There were no immediately apparent hyperlinks or wormholes that connect to it, but there are still unusual energy fluctuations in the area. Unusual even for the Ember Waste." He looks up at you. "We are hoping that you might be able to figure out a way in."`

Reviewer: ```suggestion
			`	Chilia notices you eyeing the deviations. "Perceptive, captain. Our sensors have reported some unusual energy signatures coming from this location. Scouts in the area say they are picking up signs of a new system. There were no immediately apparent hyperlinks or wormholes that connect to it, but there are still unusual energy fluctuations in the area. Unusual even for the Ember Waste." He looks up at you. "We are hoping that you might be able to figure out a way in."`
```

			choice
				`	"I could see if I can reach it with a Jump Drive."`
				`	"I will see if I can find another way."`
			`	"Thank you, Captain <first>. We have some ships that can follow, and there are other scouts already in the area that should be reporting soon. Given your experience with the wider galaxy, though, we would like you to lead the way."`
				accept
	on enter "Postverta"
		dialog

> 			`As the sensors re-establish their depiction of the system around you, an alien station immiediately becomes apparent. It appears to be old and mostly inactive, but your sensors do pick up ocasional bursts of energy. Oddly, your sensors are reporting faint signs of life almost everywhere.`

Reviewer: ```suggestion
			`As your ship's sensors re-establish their depiction of the system around you, an alien space station comes into view. It appears to be old and mostly inactive, but your sensors do pick up occasional bursts of energy - and, oddly, faint signs of life almost everywhere.`
```

	on stopover
		dialog

> 			`The station is surprisingly still active, despite its apparent age. You poke around a bit, but its mysteries remain just that.`

Reviewer: I feel like we need a bit more description than this. :) What's it like inside the alien space station?

Author: Significantly expanded the detail for this mission.

	on complete
		dialog

> 			`As you arrive back at Viminal you broadcast a warning to the port authorities that you have something following you. You inform them that it is not hostile, but it is an unknown that originated in the new system.`

Reviewer: ```suggestion
			`As you arrive back at Viminal, you broadcast a warning to the port authorities that something is following you. You inform them that it is not hostile, but it is an unknown entity that originated in the new system.`
```

	landing
	source "Viminal"
	waypoint
		system Peragenor
		system Convector
		system Nenia
	destination Nasqueron
	to offer
		has "Remnant: Cognizance 12: done"
	on offer
		conversation
			`Quite a number of Remnants have gathered around the Spaceport, watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`
			`	After several hours of watching the creature, one captain pushes his way forward to where you, Chilia, and Plume are talking. "I think I have an idea about this thing, Prefect!" the man signs excitedly. "Look at the positions of the glowing tips when it freezes. That pattern there matches the star positions of the main Ember Waste systems when viewed from Peragenor, and that second matches Convector." He pulls out a datapad and pulls up a projection of the starmap. "It is hard to tell due to the imprecision, but it looks like the third freeze is probably Antevorta or Nenia."`

> 			`	Plume chimes in "If those freezes indicate star positions, then the flying and landing behavior in between them may be indicating it wants you to go. Please give it a try."`

Reviewer: ```suggestion
			`	Plume chimes in "If those freezes indicate star positions, then the flying and landing behavior in between them may be indicating that it wants you to travel there. Please give it a try."`
```

				accept
	npc accompany
		government "Ember Waste"
		personality mute
		ship "Ember Waste Node" "Unknown Entity"
	on enter Peragenor
		dialog `The creature comes to a stop in front of you, spins around briefly, then spreads out its tentacles in another pattern. Following the Remnant's intuition, you match it to your star charts as probably being Convector. After holding the position for a minute, the creature realigns itself and starts to shimmer with the tell-tale signs of starting to make a jump, but it holds it, as if waiting for you.`
	on enter Convector
		dialog `It seems the analysis of the location was right. The creature quickly approaches you and repeats its previous dance, ending up with a slightly different frozen position. From this vantage, you suspect that it is probably Nenia. This time, as you watch it more closely, it looks like it is following in your wake somehow.`
	on enter Nenia

> 		dialog `You have never seen such activity from the Void Sprites. Many quickly congregate around the strange ship, and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath ship, presumably another Raider.`

Reviewer: ```suggestion
		dialog `You have never seen such activity from the Void Sprites. Many quickly congregate around the strange ship, and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath Raider.`
```

			choice
				`	"How are we supposed to lure them inside?"`
				`	"Are we sure this is a good idea?"`
					goto goodidea
				`	"Okay, let's go do this."`
					goto letsgo
			`	Plume looks a bit embarrassed. "Centuries ago when we first studied the void sprites, we learned how to harvest them. As part of that, we discovered that they were particularly interested in certain types of radiation. Likewise, if they discovered a certain kind of element common in some asteroids, they would ignore almost everything while they chewed on it. So, we have emitters inside the cargo holds, along with asteroids to keep them busy for a short while."`
			choice
				`	"Are we sure this is a good idea?"`
				`	"Okay, let's go."`
					goto letsgo
			label goodidea
			`	Chilia signs with a resolute tone. "Is it a good idea? Maybe, maybe not. Is it the best idea we have so far? Yes." He sighs, "I fully expect the Archon to interfere: they do have long memories. As best we can tell, they can pick out who is leading a fleet, so you should expect to have them talking to you. That is, if they do not simply start shooting, anyway."`
			label letsgo

> 			`	Chilia continues "You don't need to actually land anywhere in Nenia this time, Captain. Just make sure the Pelicans get there, loadup, and return safely." The two stand up and get ready to leave. "May the Embers burn bright for you, Captain." With that, they disembark and quickly clear the blast area around the landing pad. Around you a large group of pelicans are warming up, each one connecting in to your IFF and confirming their status."`

Reviewer: ```suggestion
			`	Chilia continues "You don't need to actually land anywhere in Nenia this time, Captain. Just make sure the Pelicans get there, loadup, and return safely." The two stand up and get ready to leave. "May the Embers burn bright for you, Captain." With that, they disembark and quickly clear the blast area around the landing pad. Around you a large group of pelicans are warming up, each one synchronizing to your IFF and confirming their status."`
```

				accept
	npc save accompany
		government "Remnant"
		personality pacifist coward launching escort
		fleet
			names "remnant"
			variant
				"Pelican (Heavy Lift)" 6
	npc
		government "Remnant"
		personality launching escort disables plunders
		fleet "Large Remnant" 3
	npc evade
		government "Korath"
		personality heroic disables plunders harvests entering staying
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath World-Ship B (Ember)"
				"Korath Raider (Ember)" 3
	on enter "Nenia"

> 		dialog `As you enter the system, you quickly search for the Archon, but it is nowhere to be seen. The Korath hunting party, however, is clearly waiting for you and moves to attack. As they do so, the pelicans immediatley set to work collecting void sprites. As the last one closes its hold again, the confirmation codes flash across your tactical screen. "Good to go."`

Reviewer: ```suggestion
		dialog `As you enter the system, you quickly search for the Archon, but it is nowhere to be seen. A Korath hunting party, however, is clearly waiting for you and moves to attack. As their ships converge on you, the pelicans immediately set to work collecting void sprites. As the last one closes its hold again, a confirmation codes flash across your tactical screen. "Good to go."`
```

	on enter "Pantica"
		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into Esquilion's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`
	on complete
		log "Succesfully introduced a flock of void sprites to Esquiline. Time will tell how they fare."
		"salary: Remnant" = 2500
		payment 1000000
		event "remnant: pantica void sprites"
		event "remnant: cognizance timer" 30
		conversation

> 			`	Moments later a ping comes in on the comlink, and Prefect Chilia and Plume appear onscreen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in" concludes Plume.`

Reviewer: ```suggestion
			`	Moments later a ping comes in on the comlink, and Prefect Chilia and Plume appear onscreen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
```

> 			`	"In the meantime," continues Chilia. "We are allocating you a daily stipend from our resources to cover some of your ongoing crew, maintnance, and upgrading costs. A larger one time allotment has already been credited to your account with the shipyard facilities to help cover costs incurred in this mission."`

Reviewer: ```suggestion
			`	"In the meantime," continues Chilia. "We are allocating you a daily stipend from our resources to cover some of your ongoing crew, maintenance, and upgrading costs. A larger one-time allotment has already been credited to your account with the shipyard facilities to help cover costs incurred during this mission."`
```

event "remnant: pantica void sprites"
	system "Pantica"
		add fleet "Void Sprites" 4000
	planet "Esquiline"
		government "Remnant"
		add attributes remnant
		remove attributes uninhabited
		description `While Esquiline used to be simply another planet the Remnant harvested valuable resources from, the introduction of void sprites and the activation of their station has turned this world into a hidden research lab: The first human astrobiology station.`

> 		add spaceport `	Although the facilities here are basic, the Remnant cafeteria boasts a food dispenser that is available at any time; along with a mesmerizing view of the clouds outside. Occasionally a void sprite will pass in front of the window, much to the delight of everyone.`

Reviewer: ```suggestion
		add spaceport `	Although the facilities here are basic, the Remnant cafeteria boasts a food dispenser that is available at any time, along with a mesmerizing view of the clouds outside. Occasionally a void sprite will pass in front of the window, much to the delight of everyone.`
```

				accept
	npc
		government "Drak"
		system "Nenia"
		personality staying heroic disables uninterested
		ship "Archon (Cloaked)" "Lifted Lorax"
	npc
		government "Korath"
		system "Nenia"
		personality entering forbearing timid uninterested surveillance staying
		ship "Korath Raider" "Korath Raider (Ember)"
			cargo "void sprite" 5 20
	on enter "Nenia"
		dialog
			`As you scan the system the first thing you notice is that the void sprites' behavior seems different: There are fewer of them flying around, and the few that you see seem to be almost timid - floating quietly, and disappearing into the clouds whenever your ship moves towards them.`
			`	After several minutes of recording the illusive void sprites you decide to try on a gas giant, recalling that Plume will want footage from both giants.`
	on stopover
		conversation
			`Here you find the same behaviour as before: Void sprites that have suddenly become timid and prone to diving into the shrouded depths of their world at the slightest quick movement.`
			`	Despite the new challenge, you manage to record a variety of different void sprites before deciding to head back to <destination>.`
	on complete
		conversation

> 			`Disappointed in the void sprites' behavior, you deliver the recordings to Plume's lab without fanfare. He quickly takes them and reviews the footage. He makes a gesture that you think means 'curious dejection,' but then seems to notice something and resumes his usual intent bearing.`

Author: As in, "He was intent on the screen", as in focused on it.

			`	This is one of the aspects that set prefects apart: They are Remnant who have demonstrated excellence in their field, sound judgment across a variety of fields, and an affinity for balancing a multitude of demands on Remnant resources. Humility is also a requirement, as they must always recognize that while their particular combination of skills may be unique, there is always someone else who is better than them at any one thing. As such, Prefects are the adjudicators in determining the priority of our various tasks, and the leaders who keep the Remnant unified.`
			choice
				`	"Thanks for the explanation!"`
					goto acceptance
				`	"I thought prefects were specialists?"`
			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the information and suggestions from other prefects and especially the specialists in the field or fields relevant to the task.`

> 			`	"Given your own skills, I suspect that the prefects you are most likely to interact with would be Prefect Chilia who has the task of co-ordinating the military defense of the Remnant; and Prefect Taely who has the task of integrating our latest innovations into usable ships."`

Reviewer 2: ```suggestion
			`	"Given your own skills, I suspect that the prefects you are most likely to interact with would be Prefect Chilia, coordinates the military defense of the Remnant, and Prefect Taely, who integrates our latest innovations into usable ships."`
```
No hyphen in `coordinate` is preferred American style.

			choice
				`	"Thanks! That was more detail than I was expecting."`
				`	"Okay, that satisfied my curiosity for now."`
			label acceptance

> 			`	"Excellent. You deal with the Korath, and I will file a report about this threat. Please inform me immediately once you have dealt with the threat." He pauses, then continues, "Oh, and remember to record the battle. It may be useful to help understand what they are doing there."`

Reviewer 2: ```suggestion
			`	"Excellent. You deal with the Korath, and I will file a report about this threat. Please inform me immediately once you have taken care of it." He pauses, then continues, "Oh, and remember to record the battle. It may be useful in understanding what they are doing there."`
```
`Dealt with the threat` felt repetitive since it was used in the previous sentence. Other part was just optional, but I thought it sounded a bit more natural.

				accept
	npc kill
		government "Korath"
		system "Nenia"
		personality forbearing timid uninterested surveillance staying
		ship "Korath Raider" "Korath Raider (Ember)"
			cargo "void sprite" 5 20
		dialog "You have destroyed the Korath Vessel. You can now return to <planet> to collect your payment."
	on complete
		event "korath hunting"
		conversation

> 			`When you return to Aventine, you find Plume waiting for you. He eagerly reviews the recording of the combat. "Thank you for eliminating this hunter..." he is interrupted by a flashing light on his commlink. He briefly reads something, then looks back up to you. "Apparently something about the Korath hunting the void sprites has concerned the defence prefect. Chilia has requested that you meet with him on Caelian immediately."`

Reviewer 2: ```suggestion
			`When you return to Aventine, you find Plume waiting for you. He eagerly reviews the recording of the combat. "Thank you for eliminating this hunter-" he is interrupted by a flashing light on his commlink. He briefly reads something then looks back up to you. "Apparently something about the Korath hunting the void sprites has concerned the defense prefect. Chilia has requested that you meet with him on Caelian immediately."`
```
I like a dash instead of an ellipsis to signify interruption, but it's personal preference.

event "korath hunting"
	government "Korath"
		"attitude toward"
			"Indigenous Lifeform" -.01
			"Ember Waste" -.01

> 

Author: OK, I have such a mission added.

mission "Remnant: Cognizance 3"
	name "Search for Korath"
	description "Prefect Chilia has asked you to track down a Korath fleet"
	source "Caelian"
	landing
	waypoint
		system "Aescolanus"
	to offer
		has "Remnant: Cognizance 2: done"
	on offer

> 		log "People" "Chilia" `Prefect Chilia is, compared to most military leaders you have met, an unusual person. He is inclined to do things himself, rarely standing on protocol, if at all.`

Reviewer 2: People logs are all written in third-person, not second (had to check to make sure, but this does seem to be the norm).

Author: Fixed, and relocated to an external mission that relies on the incremented chilia variable to trigger.

			`	"Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles deprecatingly. "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them.`
				goto main
			label chilia
			`The <ship> has barely come to a rest on its designated pad when your panel is blinking with a request for permission to come aboard. A quick glance at the viewscreen shows Prefect Chilia standing at your airlock. You quickly hit the unlock button and watch as he strides onboard.`
			`	"Greetings, Captain <first>," he sings. "You seem to have a knack for finding things of importance to us. I am glad, however, that you and Plume discovered this."`
			label main
			branch voidrifle
				has "Remnant: Face to Maw 2B: done"

> 			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites, and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. There are military implications of the Korath harvesting the void sprites, and we must take steps to prevent this from happening.`

Reviewer 2: ```suggestion
			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. There are military implications of the Korath harvesting the void sprites, and we must take steps to prevent this from happening.`
```

				goto next
			label voidrifle

> 			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites, and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. You are aware of the potential military implications of the Korath harvesting the void sprites, and must realize the importance of stopping such discovery.`

Reviewer 2: ```suggestion
			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites, and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. You are aware of the potential military implications of the Korath harvesting the void sprites and must realize the importance of stopping such discovery.`
```

			label next

> 			`	 "It is imperative that all the Korath involved in this exploration be located immediately. According to our surveillance, that ship was part of a fleet that recently passed through Parca. Locate the fleet, and report back."`

Reviewer 2: ```suggestion
			`	 "It is imperative that all the Korath involved in this exploration be located immediately. According to our surveillance, that ship was part of a fleet that recently passed through Parca. Locate the fleet and report back."`
```

				accept
	npc
		government "Korath"
		personality staying surveillance
		system "Aescolanus"
		fleet
			names "korath"
			variant
				"Korath World-Ship B (Ember)"
				"Korath Raider (Ember)" 4
	on enter "Aescolanus"
		dialog `You have located the Korath fleet. They do not seem to be going anywhere at the moment.`
	on complete
		payment 500000
		conversation

> 			`This time as you complete your post-flight checks you spot Prefect Chilia exiting the starling next to you and watch as he strides across the pad to your airlock. You hit the open button as he clambers up the ladder. Without a word you pull up the sensor logs from Aescolanus, highlighting the position of the various ships. His eyes flit over the display for a moment, then he nods.`

Reviewer 2: ```suggestion
			`This time, as you complete your post-flight checks you spot Prefect Chilia exiting the starling next to you and watch as he strides across the pad to your airlock. You hit the open button as he clambers up the ladder. Without a word you pull up the sensor logs from Aescolanus, highlighting the position of the various ships. His eyes flit over the display for a moment, then he nods.`
```

mission "Remnant: Cognizance 8"
	name "Transport engineer to Esquiline"
	description "An engineer working for Plume needs transportation for himself and some equipment to Esquiline."
	blocked "Plume's engineer needs a free bunk and 8 tons of cargo space for his equipment. Come back when you have enough space."
	cargo equipment 8
	passengers 1
	source "Aventine"
	destination "Esquiline"
	to offer
		has "Remnant: Cognizance 7: done"
	on offer
		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed esquiline station.`
		conversation
			`You walk into Plume's lab and are greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`

> 			`	"It will be a pleasure to work with you," sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`

Author: Yes. Good catch.

	destination "Viminal"
	to offer
		has "Remnant: Cognizance 9: done"
	on offer
		conversation
			`As you finish unloading, you spot Gavriil approaching. He gestures in greeting as he approaches. "Thanks for the delivery. We've got standard logistics routes set up now for the rest."  He pauses and fishes a datacube out of a pouch. "Could you take this to Prefect Taely on Viminal? She is overseeing some retrofits and will need this data.`
				accept
	on complete
		conversation
			`On Viminal you easily locate the maintenance yards and eventually find Taely buried in a Pelican. She spots you approaching and frees herself from the machinery. "Greetings <first>. How is our station faring?" You reply that it is improving rapidly, and hand her the datacube. "Ah, good. The detailed chemical analysis and profiles." She slots it into her console and taps a few commands. "Okay, I just need to incorporate these profiles into the system and make a few more adjustments. Meet me in the cafeteria in a couple hours."`

mission "Remnant: Cognizance 11"
	name "Travel with the <escort target ship> to pick up a shipment of atmosphere from Nasqueron and deliver it to <destination>."

> 	description "Go meet up with escorts"

Author: Looks like it. Fixed, thanks!

	on complete
		event "remnant: postverta reveal"
		conversation
			`You arrive at the hidden station on Esquiline, where Gavriil is waiting to offload the atmospheric gasses. His team quickly hooks the puffin up to ports on the bay walls, and hidden equipment starts pumping its contents into the system. After a few minutes, they disconnect the ships, and you and the puffins head up to the pelican to refill your reservoirs and ferry it down to the station.`
			`	As the last load starts to empty into the station another tech comes up to you. "Captain, we just received a message from Prefect Chilia. He would like you to meet him on Viminal as soon as possible." You nod and start to run checks on the <ship>, making ready to leave once the cargo is done unloading.` 

> 

Author: Added a small mission just to give the player an entry in their mission list and a flag so they will know to return to Viminal.

			distance 1408
			period 554
			sprite planet/sheragi_postverta
		object
			distance 1208
			period 654
			sprite planet/rock10
		object
			distance 439
			period 143
			sprite planet/desert7
		object
			distance 814
			period 362
			sprite planet/fog0
			object
				distance 202
				period 21
				sprite planet/dust4
		object
			period 19
			sprite star/m4
		pos 297 269
	unvisit Levana
	unvisit Caeculus
	unvisit Prosa
	unvisit Egeria
	system Levana
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Caeculus
		add object wormhole_barren2
			distance 2400
			period 391
			sprite planet/wormhole-red
	system Prosa
		add object wormhole_barren2
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Egeria
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
		

mission "Remnant: Cognizance 12"
	name "Investigate strange sensor readings"
	description "Long range sensors have picked up strange gravitational waves. Go investigate."
	landing
	source "Viminal"
	waypoint
		system Postverta
	stopover
		planet "Ssil Vida"
	to offer
		has "Remnant: Cognizance 11: done"
	on offer
		conversation

> 			`You land by the starport and are immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship and escorts you to a nearby starling. Inside you find Chilia and several other Remnants clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match the star maps you're familiar with.`

Author: The plural of Remnant is... Remnant. Remnants just doesn't quite sound right.

			`	The engineering of the station points to a dedicated construction program aimed at a single purpose. You trace some of the more obvious set of cables and discover a large spherical chamber filled with machinery. Unfortunately, it is still emiting significant amounts of energy - enough that your suit's medical systems emphatically warn you to turn back. So while the large sphere appears to be central to this station's purpose, it will have to remain a mystery for now. Clearly, this is the sort of thing that the Remnant sent you out to find.`
	on complete
		dialog
			`As you arrive back at Viminal, you broadcast a warning to the port authorities that something is following you. You inform them that it is not hostile, but it is an unknown entity that originated in the new system.`

mission "Remnant: Cognizance 12b"
	landing
	invisible
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 12: active"

> 	npc accompany

Author: It does have a cloak, so it is perfectly capable of doing so. So long as it is sufficiently visible that the player can tell it is following them, things are good.

	on enter Nenia
		dialog `You have never seen such activity from the Void Sprites. Many quickly congregate around the strange ship, and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath Raider.`
	on complete
		event "remnant: nenia ember waste node"
		conversation
			`You follow the creature down through the clouds of Nasqueron and emerge into what can only be described as an amphitheater in the clouds. Thousands of void sprites fill the area and the creature you have been following floats near the center of it. Your external sensors pick up myriad pulses of energy and eerie whistling noises that seem to resonate in the clouds. As the <ship> moves forward there is a sudden silence and the flickers of energy synchronize.`
			`	You feel a sudden pressure in your mind, and a sense of presence as the world around you fades out somewhat.`

> 			`	"Know you not, do I. Live creature inside not-quite creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - perhaps I shall hear."`

Author: Made them uniformly capitalized. Thanks!

			`	With that, the pressure vanishes and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are quite sure that the Remnant are going to want to hear about this.`

> 

Reviewer 2: Same here with adding a mission to ensure players return to Remnant space.

Author: Done!

			choice
				`	"Are we sure this is a good idea?"`
				`	"Okay, let's go."`
					goto letsgo
			label goodidea
			`	Chilia signs with a resolute tone. "Is it a good idea? Maybe, maybe not. Is it the best idea we have so far? Yes." He sighs, "I fully expect the Archon to interfere: they do have long memories. As best we can tell, they can pick out who is leading a fleet, so you should expect to have them talking to you. That is, if they do not simply start shooting, anyway."`
			label letsgo
			`	Chilia continues "You don't need to actually land anywhere in Nenia this time, Captain. Just make sure the Pelicans get there, loadup, and return safely." The two stand up and get ready to leave. "May the Embers burn bright for you, Captain." With that, they disembark and quickly clear the blast area around the landing pad. Around you a large group of pelicans are warming up, each one synchronizing to your IFF and confirming their status."`
				accept

> 	npc save accompany

Reviewer 2: In my playtest, one of the Albatrosses supposed to accompany you wouldn't go past the Edusa system. My guess is it's missing a Quantum Keystone.

Author: All Remnant ships should have Quantum Key Stones now, thanks to #5202 . Three Albatross variants and three gull variants didn't have Key Stones. Now they do. So this should be fixed now.

	on complete
		log "Succesfully introduced a flock of void sprites to Esquiline. Time will tell how they fare."
		"salary: Remnant" = 2500
		payment 1000000
		event "remnant: pantica void sprites"
		event "remnant: cognizance timer" 30
		conversation
			`	Moments later a ping comes in on the comlink, and Prefect Chilia and Plume appear onscreen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
			`	"In the meantime," continues Chilia. "We are allocating you a daily stipend from our resources to cover some of your ongoing crew, maintenance, and upgrading costs. A larger one-time allotment has already been credited to your account with the shipyard facilities to help cover costs incurred during this mission."`

event "remnant: pantica void sprites"
	system "Pantica"
		add fleet "Void Sprites" 4000
	planet "Esquiline"
		government "Remnant"
		add attributes remnant
		remove attributes uninhabited

> 		description `While Esquiline used to be simply another planet the Remnant harvested valuable resources from, the introduction of void sprites and the activation of their station has turned this world into a hidden research lab: The first human astrobiology station.`

Author: Changed the conversation slightly. So now it is " The first human astrobiology station studying void sprites." Because even with additional missions, I strongly doubt that any future mission chains could have humans building a cloud-giant space station in Nenia before this point in the Remnant story. (Actually I strongly doubt that it would make sense to ever have it happen, but it should definitely be much later in the game than the Remnant stuff.)

> 		add spaceport `	Although the facilities here are basic, the Remnant cafeteria boasts a food dispenser that is available at any time, along with a mesmerizing view of the clouds outside. Occasionally a void sprite will pass in front of the window, much to the delight of everyone.`

Author: There is a space between spaceport and the first backtick, so I think it is laid out correctly.

> 		add spaceport `	Although the facilities here are basic, the Remnant cafeteria boasts a food dispenser that is available at any time, along with a mesmerizing view of the clouds outside. Occasionally a void sprite will pass in front of the window, much to the delight of everyone.`

Reviewer 2: (Whoops, didn't see this until just now.) I was talking about the space between the backtick and "Although", not the one between "spaceport" and the backtick.

			`	"Greetings, Captain <first>," he sings. "You seem to have a knack for finding things of importance to us. I am glad, however, that you and Plume discovered this."`
			label main
			branch voidrifle
				has "Remnant: Face to Maw 2B: done"
			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. There are military implications of the Korath harvesting the void sprites, and we must take steps to prevent this from happening.`
				goto next
			label voidrifle
			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites, and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. You are aware of the potential military implications of the Korath harvesting the void sprites and must realize the importance of stopping such discovery.`
			label next

> 			`	 "It is imperative that all the Korath involved in this exploration be located immediately. According to our surveillance, that ship was part of a fleet that recently passed through Parca. Locate the fleet and report back."`

Author: Hmmm, this is still a quotation, so it should continue to have the quotation marks at the start. If I recall the rule correctly, if a quotation carries over a paragraph break the first paragraph doesn't have a quotation mark at the end, but the next paragraph *does* have a quotation mark at the start to show that it is still a quotation.

				accept

> 	npc

Author: Added alternate dialog for if the player destroys the fleet to Cognizance 4; as well as a branch to the dialog of Cognizance 5.

			`	"Know you not, do I. Live creature inside not-quite creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - Perhaps I shall hear."`
			`	With that, the pressure vanishes, and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are quite sure that the Remnant are going to want to hear about this.`

event "remnant: nenia ember waste node"
	system "Nenia"
		add fleet "Ember Waste Node" 10000
	

mission "Remnant: Cognizance 17"
	name "Report back to Aventine."
	description "The Remnant will want to know about what happened on Nasqueron."
	source "Nasqueron"
	landing
	destination "Aventine"
	to offer
		has "Remnant: Cognizance 16: done"
	

mission "Remnant: Cognizance 18"
	name "Go collect some void sprites"
	description "Escort the Pelican fleet to <waypoints> and back to <planet>."
	source
		government "Remnant"
	landing

> 	destination Aventine

Reviewer 3: ```suggestion
	destination Esquiline
```
This should be Esquiline, right?

Author: No, because to land on Esquiline the player would need to be flying something with gaslining. The mission explicitly tells the player they don't need to land in Nenia as a subtle hint they should probably fly something more combative than the puffin or (if unlocked, penguin) The player doesn't need to actually go to Esquiline, just Pantica (which is the system Esquiline is in); and then proceed to land on Aventine.

Reviewer 3: Makes sense

				accept
	npc save accompany
		government "Remnant"
		personality pacifist coward launching escort
		fleet
			names "remnant"
			variant
				"Pelican (Heavy Lift)" 6
	npc
		government "Remnant"
		personality launching escort disables plunders
		fleet "Large Remnant" 3
	npc evade
		government "Korath"
		personality heroic disables plunders harvests entering staying
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath World-Ship B (Ember)"
				"Korath Raider (Ember)" 3
	on enter "Nenia"
		dialog `As you enter the system, you quickly search for the Archon, but it is nowhere to be seen. A Korath hunting party, however, is clearly waiting for you and moves to attack. As their ships converge on you, the Pelicans immediately set to work collecting void sprites. As the last one closes its hold, the confirmation codes flash across your tactical screen. "Good to go."`
	on enter "Pantica"

> 		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into Esquilion's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later, you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`

Reviewer 3: ```suggestion
		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into Esquiline's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later, you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`
```
If the destination was supposed to be Esquiline don't use this one, use the other suggestion

> 		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into Esquilion's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later, you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`

Reviewer 3: ```suggestion
		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into <planet>'s atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later, you receive a message from <planet> station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`
```
Use this suggestion only if the destination is Esquiline

Author: It isn't. This is a description of the player watching from orbit, *not* landing on Esquiline themselves.

			`	"All right, enough history." He turns to the technicians who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
			scene "outfit/void rifle"

mission "Remnant: Cognizance 1"
	name "Follow-up Observations"
	description "Plume would like you to do a follow-up visit to the planets in <system>."
	minor
	source "Aventine"
	stopover "Nasqueron"
	stopover "Slylandro"
	cargo "scanning equipment" 8

> 	blocked "The Remnant researcher has another job for you, but you're going to have to free up some cargo space in order to take it on."

Author: Fixed. Thanks!

			choice
				`	"Thanks for the explanation!"`
					goto acceptance
				`	"I thought prefects were specialists?"`
			`	"Well, they are and are not. Most prefects are assigned particular tasks that they are expected to devote themselves to. Often, this task relates to their area of expertise, but it is not always the case. Generally speaking, a prefect is considered to be the ultimate authority on tasks that they have been assigned to, but they are expected to take into account the suggestions from other prefects and the specialists in the fields relevant to the task.`
			`	"Given your own skills, I suspect that the prefects you are most likely to interact with would be Prefect Chilia, coordinates the military defense of the Remnant, and Prefect Taely, who integrates our latest innovations into usable ships."`
			choice
				`	"Thanks! That was more detail than I was expecting."`
				`	"Okay, that satisfied my curiosity for now."`
			label acceptance

> 			`	"Excellent. You deal with the Korath, and I will file a report about this threat. Please inform me immediately once you have taken care of it." He pauses, then continues, "Oh, and remember to record the battle. It may be useful in understanding what they are doing there."`

Author: Thanks for the feedback. I have added in a small `on enter Nenia` where the pilot activates the recording so the player shouldn't be left hanging thinking they need to do something specific.

> 			`	"Excellent. You deal with the Korath, and I will file a report about this threat. Please inform me immediately once you have taken care of it." He pauses, then continues, "Oh, and remember to record the battle. It may be useful in understanding what they are doing there."`

Reviewer: Thanks!

			period 391
			sprite planet/wormhole-red
	system Prosa
		link Postverta
		link Vaticanus
		add object wormhole_barren2
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Egeria
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
		

mission "Remnant: Cognizance 14"
	name "Investigate strange sensor readings"
	description "Long range sensors have picked up strange gravitational waves. Go investigate, then return to <destination>."
	landing
	source "Viminal"
	waypoint
		system Postverta
	stopover
		planet "Ssil Vida"
	to offer
		has "Remnant: Cognizance 13: done"
	on offer
		conversation
			`You land by the starport and are immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship and escorts you to a nearby Starling. Inside you find Chilia and several other Remnant clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match the star maps you're familiar with.`

> 			`	Chilia notices you eyeing the deviations. "Perceptive, Captain. Our sensors have reported some unusual energy signatures coming from this location. Scouts in the area say they are picking up signs of a new system. There were no immediately apparent hyperlinks or wormholes that connect to it, but there are still unusual energy fluctuations in the area. Unusual even for the Ember Waste." He looks up at you. "We are hoping that you might be able to figure out a way in."`

Author: Thanks for the feedback. Now that you've pointed it out, I realize that I share the same dislike - although possibly not as strongly. 

In this particular case it seems like a good way to carry the conversation/plot forward; but I will be careful of using things like this in the future.

> 			`	Chilia notices you eyeing the deviations. "Perceptive, Captain. Our sensors have reported some unusual energy signatures coming from this location. Scouts in the area say they are picking up signs of a new system. There were no immediately apparent hyperlinks or wormholes that connect to it, but there are still unusual energy fluctuations in the area. Unusual even for the Ember Waste." He looks up at you. "We are hoping that you might be able to figure out a way in."`

Reviewer: All right.

	on complete
		dialog
			`As you arrive back at Viminal, you broadcast a warning to the port authorities that something is following you. You inform them that it is not hostile, but it is an unknown entity that originated in the new system.`

mission "Remnant: Cognizance 15"
	landing
	invisible
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 14: active"
	npc accompany
		government "Ember Waste"
		personality waiting mute
		ship "Ember Waste Node" "Unknown Entity"
	

mission "Remnant: Cognizance 16"
	name "Follow the Creature"
	description "The creature seems to be creating constellations and trying to get you to follow it."
	landing
	source "Viminal"
	waypoint
		system Peragenor
		system Convector
		system Nenia
	destination Nasqueron
	to offer
		has "Remnant: Cognizance 14: done"
	on offer
		conversation

> 			`Quite a number of Remnant have gathered around the spaceport watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`

Reviewer: ```suggestion
			`Quite a number of Remnant have gathered around the spaceport watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries around the area have their formidable-looking weapons trained on the creature in case it turns violent.`
```

			`	Plume chimes in, "If those freezes indicate star positions, then the flying and landing behavior in between them may be indicating that it wants you to travel there. Please give it a try."`
				accept
	npc accompany
		government "Ember Waste"
		personality mute
		ship "Ember Waste Node" "Unknown Entity"
	on enter Peragenor
		dialog `The creature comes to a stop in front of you, spins around briefly, then spreads out its tentacles in another pattern. Following the Remnant's intuition, you match it to your star charts as probably being Convector. After holding the position for a minute, the creature realigns itself and starts to shimmer with the tell-tale signs of starting to make a jump, but it holds it, as if waiting for you.`
	on enter Convector
		dialog `It seems the analysis of the location was right. The creature quickly approaches you and repeats its previous dance, ending up with a slightly different frozen position. From this vantage, you suspect that it is probably Nenia. This time, as you watch it more closely, it looks like it is following in your wake somehow.`
	on enter Nenia

> 		dialog `You have never seen such activity from the void sprites. Many quickly congregate around the strange ship and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath Raider.`

Reviewer: ```suggestion
		dialog `You have never seen such activity from the void sprites. Many quickly congregate around the strange creature that has been guiding you, and escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath Raider.`
```

			`	With that, the pressure vanishes, and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are certain that the Remnant are going to want to hear about this.`

event "remnant: nenia ember waste node"
	system "Nenia"
		add fleet "Ember Waste Node" 10000
	

mission "Remnant: Cognizance 17"
	name "Report back to Aventine."
	description "The Remnant will want to know about what happened on Nasqueron."
	source "Nasqueron"
	landing
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 16: done"
	

mission "Remnant: Cognizance 18"
	name "Go collect some void sprites"
	description "Escort the Pelican fleet to <waypoints> and back to <planet>."
	source
		government "Remnant"
	landing
	destination Aventine
	waypoint "Nenia"
	to offer
		has "Remnant: Cognizance 17: done"
	on offer
		conversation

> 			`It is almost starting to feel commonplace to have Remnant pouncing on your ship and using your cockpit as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permission to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`

Reviewer: ```suggestion
			`It is almost starting to feel commonplace to have Remnant pouncing on your ship and using your bridge as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permission to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`
```

Then again I suppose "cockpit" would be more accurate if the player is flying a small interceptor type ship, but that's probably not very likely at this point in the storyline.

Author: Making a branch for this might be a possibility, but I'm not sure how important doing so would be.
